### ISSUE TITLE
Parallel error when dowloading

### ISSUE THREAD

**Issue by lshilab** [pirovc/genome_updater#50]
Hi, I tried to download the virus genome indicated in the readme file and the program runs successfully in the home folder. But when trying to download in the /mnt/d/ folder, it shows a parallel error. I need to download to the /mnt/d/ because of space limitations. Can you check if there is something wrong with my settings? Thanks!

-------------------------------------------
Mode: NEW - DOWNLOAD
Working directory: /mnt/d/virus/all_rna_virus/
-------------------------------------------
Downloading assembly summary [2022-04-16_08-25-17]
 - Database [refseq]
 - 257586 assembly entries available
 - Downloading new taxdump and parsing lineages
 - 253246 assemblies removed not in taxids [2559587]
 - 4340 assembly entries to download

 - Downloading 4340 files with 12 threads
**parallel: This should not happen. You have found a bug. Please follow**
https://www.gnu.org/software/parallel/man.html#REPORTING-BUGS

Include this in the report:
* The version number: 20220222
* The bugid: Can't dup STDOUT: No such file or directory
* The command line being run
* The files being read (put the files on a webserver if they are big)

If you get the error on smaller/fewer files, please include those instead.

**Comment by pirovc** [comment]
Looks like it's a parallel issue. In genome_updater, parallel is set to use temporary directory, same as the output directory. You may be able to work around the problem by changing it [here](https://github.com/pirovc/genome_updater/blob/a5df18f2759e0778810729c1fabb065263d5a9ef/genome_updater.sh#L454). Alternatively you could try using a different parallel version, maybe the one distributed by conda?

**Comment by lshilab** [comment]
Thank you for pointing out this solution, I think it's quite possible that this will solve the problem. Unfortunately, I am just starting to learn and don't quite understand exactly what to do with the change you are pointing about. I tried to bypass this problem by changing the output folder. I was using the script under wsl, and when I couldn't run it, the output folder was not in wsl, but on another disk. I tried putting the output folder in the home directory and was able to download normally. I guess this is exactly the folder problem you pointed out. Thanks again.
